developing/structure/transaction.md

```diff
@@ -0,0 +1,1152 @@
+                "value": "osmo1ejaswj8lcyh0zgnes8zt45e0w7tqm4mrxr74sfwgpdh72shp58ms4fdqsk"
+              },
+              {
+                "key": "amount",
+                "value": "3802225ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2"
+              },
+              {
+                "key": "recipient",
+                "value": "osmo1mw0ac6rwlp5r8wapwk3zs6g29h8fcscxqakdzw9emkne6c8wjp9q0t3v8t"
+              },
+              {
+                "key": "sender",
+                "value": "osmo1l4u56l7cvx8n0n6c7w650k02vz67qudjlcut89"
+              },
+              {
+                "key": "amount",
+                "value": "3802225ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2"
+              },
+              {
+                "key": "recipient",
+                "value": "osmo1l4u56l7cvx8n0n6c7w650k02vz67qudjlcut89"
+              },
+              {
+                "key": "sender",
+                "value": "osmo1mw0ac6rwlp5r8wapwk3zs6g29h8fcscxqakdzw9emkne6c8wjp9q0t3v8t"
+              },
+              {
+                "key": "amount",
+                "value": "15003605uosmo"
+              }
+            ]
+          }
+```
+
+With 18 entires, each group of 3 makes 6 transfers, which aligns perfectly with the 6 `coin_spent` and 6 `coin_received` pairs.
+
+Essentially, the original message was: "take 15000000uosmo, and using pools 1, 604, and 611, swap them.
+
+The result was 6 transfers, represented by the 6 `coin_received` and 6 `coin_spent`, or, conversely, 6 `transfer`:
+
+| From | To | Amount |
+| --- | --- | --- |
+| `osmo1l4u56l7cvx8n0n6c7w650k02vz67qudjlcut89` | `osmo1thscstwxp87g0ygh7le3h92f9ff4sel9y9d2eysa25p43yf43rysk7jp93` | `15000000uosmo` |
+| `osmo1thscstwxp87g0ygh7le3h92f9ff4sel9y9d2eysa25p43yf43rysk7jp93` | `osmo1l4u56l7cvx8n0n6c7w650k02vz67qudjlcut89` |  `289236271ibc/987C17B11ABC2B20019178ACE62929FE9840202CE79498E29FE8E5CB02B7C0A4` |
+| `osmo1l4u56l7cvx8n0n6c7w650k02vz67qudjlcut89` | `osmo1ejaswj8lcyh0zgnes8zt45e0w7tqm4mrxr74sfwgpdh72shp58ms4fdqsk` | `289236271ibc/987C17B11ABC2B20019178ACE62929FE9840202CE79498E29FE8E5CB02B7C0A4` |
+| `osmo1ejaswj8lcyh0zgnes8zt45e0w7tqm4mrxr74sfwgpdh72shp58ms4fdqsk` | `osmo1l4u56l7cvx8n0n6c7w650k02vz67qudjlcut89` | `3802225ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2` |
+| `osmo1l4u56l7cvx8n0n6c7w650k02vz67qudjlcut89`  | `osmo1mw0ac6rwlp5r8wapwk3zs6g29h8fcscxqakdzw9emkne6c8wjp9q0t3v8t` | `3802225ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2` |
+| `osmo1mw0ac6rwlp5r8wapwk3zs6g29h8fcscxqakdzw9emkne6c8wjp9q0t3v8t` | `osmo1l4u56l7cvx8n0n6c7w650k02vz67qudjlcut89` | `15003605uosmo` |
+
+These align precisely with the `coin_spent` and `coin_received` entries.
+
+**TBD:** What is the value of this? it looks like the original account traded 15000000uosmo for 289236271ibc, 289236271ibc for 3802225ibc, and then 3802225ibc for 15003605uosmo. What kind of transaction is this?
```

Comment:
this is an arb transaction, they gained 3605uosmo from doing so

Reply:
Aha. I will update.